debezium-api/src/main/java/io/debezium/engine/spi/OffsetCommitPolicy.java

```diff
@@ -24,6 +24,9 @@ public interface OffsetCommitPolicy {
      */
     class AlwaysCommitOffsetPolicy implements OffsetCommitPolicy {
 
+        public AlwaysCommitOffsetPolicy(Properties properties) {
+        }
+
```

Comment:
If the provided `properties` object isn't being used, then I believe this is mostly a superfluous change and we can simply not pass it as a ctor argument.

Reply:
This change stems from the DefaultTaskOffsetManager dynamically creating the OffsetCommitPolicy.  It's assuming all implementations will have a constructor with a Properties parameter.